fuzzers/utils.py

```diff
@@ -17,15 +17,35 @@
 import configparser
 import contextlib
 import os
+import re
 import shutil
 import subprocess
 import tempfile
+import yaml
```

Comment:
Needs to be seperated from other imports as others are of stdlib packages and yaml is third_party.